website/templates/project/settings.mako

```diff
@@ -71,28 +71,38 @@
                     <div class="panel-heading clearfix">
                         <h3 id="configureNode" class="panel-title">Configure ${node['node_type'].capitalize()}</h3>
                     </div>
-                    <div id="nodeCategorySettings" class="panel-body scripted">
-                        <h5>
-                            Category: <select data-bind="attr.disabled: disabled,
-                                                        options: categories,
-                                                        optionsValue: 'value',
-                                                        optionsText: 'label',
-                                                        value: selectedCategory"></select>
-                        </h5>
-                        <p data-bind="if: !disabled">
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: cancelUpdateCategory"
-                                    class="btn btn-default">Cancel</button>
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: updateCategory"
-                                    class="btn btn-primary">Change</button>
-                        </p>
-                        <span data-bind="css: messageClass, html: message"></span>
-
-                        <span data-bind="if: disabled" class="help-block">
-                            A top-level project's category cannot be changed.
-                        </span>
-                    </div>
+
+                    <div id="projectSettings" class="panel-body">
+                        <div class="form-group">
+                            <label>Category:</label>
+                            <select data-bind="attr.disabled: disabled,
+                                                         options: categories,
+                                                         optionsValue: 'value',
+                                                         optionsText: 'label',
+                                                         value: selectedCategory"></select>
+                            <span data-bind="if: disabled" class="help-block">
+                              A top-level project's category cannot be changed
+                            </span>
+                        </div>
+                        <div class="form-group">
+                            <label for="title">Title:</label>
```

Comment:
@abought  I checked and they do submit data to different routes. Dashboard posts to api/vi/project/user/edit/ whereas the settings page puts to the same minus the 'edit/'  
I appended "edit/" to the url and used postJSON instead of putJSON, but I get a 404 when I try to edit the title. 
Is there a way I can get the proper edit url?